tool/flutter_ci_script_master.sh

```diff
@@ -66,6 +65,7 @@ declare -ar PROJECT_NAMES=(
     "simplistic_editor"
     "testing_app"
     "veggieseasons"
+    "web_embedding/element_embedding_demo"
```

Comment:
@domesticmouse I think this is ready to land in `beta`. I'm not sure if this particular change is needed *now* or when beta lands into master. I guess CI will tell :)

Reply:
Can you also add this entry to stable CI? Yes it will fail, that's expected.

Reply:
I almost miss this commit. I've fixed the `flutter_ci_script_stable.sh` entry as well!